.changeset/pre.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
This file is generated when you run `yarn changeset pre <enter|exit>` and will be deleted once we publish the next stable release.